lib/arpv4.ml

```diff
@@ -103,6 +103,8 @@ module Make (Ethif : V1_LWT.ETHIF) (Clock : V1.CLOCK) (Time : V1_LWT.TIME) = str
     | Not_found ->
       Hashtbl.replace t.cache ip (Confirmed (expire, mac))
 
+  let add t ip mac = notify t ip mac
```

Comment:
the idea is to call that function on every packet. but you are right, I should probably make the name a bit more explicit and speak about timeout in the docs. And maybe expose a `add_static` function.